README.md

```diff
@@ -11,10 +11,14 @@
 
 ## Overview	
 
-Penney's game, named after its inventor Walter Penney, is a binary (head/tail) sequence generating game between two or more players. Player A selects a sequence of heads and tails (of length 3 or larger), and shows this sequence to player B. Player B then selects another sequence of heads and tails of the same length. Subsequently, a fair coin is tossed until either player A's or player B's sequence appears as a consecutive subsequence of the coin toss outcomes. The player whose sequence appears first wins [[Wikipedia](https://en.wikipedia.org/wiki/Penney%27s_game)].
-							
+Penney's game, named after its inventor Walter Penney, is a binary (head/tail) sequence generating game between two or more players. Player A selects a sequence of heads and tails (of length 3 or larger), and shows this sequence to player B. Player B then selects another sequence of heads and tails of the same length. Subsequently, a fair coin is tossed until either player A's or player B's sequence appears as a consecutive subsequence of the coin toss outcomes. The player whose sequence appears first wins.							
 Here we have a clone of this game that I wrote during coronavirus quarantine days.
 
+ℹ️ In playing with computer, the computer selects its sequence in a completely random way
```

Comment:
`In playing with the computer, the computer selects its sequence randomly`

Reply:
fixed in https://github.com/sepandhaghighi/penney/pull/67/commits/71ac75891222e7a7b415f48773eb25d7756f86ef